src/pages/Onboard/NameTeam.vue

```diff
@@ -2,25 +2,33 @@
 import { mapActions, mapGetters } from 'vuex'
 
 import { teamProfileMixin } from '@/mixins/teamProfileMixin.js'
+import { handleMembershipInvitations } from '@/mixins/membershipInvitationMixin.js'
 
 export default {
-  mixins: [teamProfileMixin],
+  mixins: [teamProfileMixin, handleMembershipInvitations],
   data() {
     return {
       // Form's computed height
       height: '0px',
 
       loading: 0,
+      pendingInvitations: [],
```

Comment:
This prop shouldn't be needed since it's already defined in the `apollo` block; queries are moved to the top level component access by default.

Reply:
nice!